packages/mathtype-ckeditor4/plugin.src.js

```diff
@@ -266,15 +267,15 @@ export class CKEditor4Integration extends IntegrationModel {
     init(editor) {
       editor.ui.addButton('ckeditor_wiris_formulaEditor', {
 
-        label: 'Insert a math equation - MathType',
+        label: StringManager.get('insert_math'),
```

Comment:
I've changed the language on the MathTypeParameters and the tooltip text on the CKEditor4 doesn't seems to change to the right language.

```js
  language: 'de',
  mathTypeParameters: {
    editorParameters: { language: 'de' }, // MathType config, including language
  },
```


![Screenshot from 2021-09-10 14-52-55](https://user-images.githubusercontent.com/85346791/132856298-4fd39abc-c9c8-465a-9c39-fd5ddc251af1.png)

Reply:
That is because CKEditor5 works very differently than the other integrations, so the change was easy and straight forward. In the other editors, the buttons labels have been changed with a dynamic strategy that technically takes the editor language to define the button label. Even so, since the specific way those integrations work, the detected language is always English, so we'll implement the necessary changes on a different task.